agents/python/opamp/http_client.py

```diff
@@ -50,14 +58,41 @@ def run(self):
         
         self.worker()
         
+    def send_first_message_disconnected(self) -> None:
+        first_disconnect_message = opamp_pb2.AgentToServer()
+        
+        agent_description = self.get_agent_description()
+        
+        first_disconnect_message.agent_description.CopyFrom(agent_description)
+        
+        agent_disconnect = self.get_agent_disconnect()
+        first_disconnect_message.agent_disconnect.CopyFrom(agent_disconnect)
+    
+        python_version = f'{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}'
+        agent_health = self.get_agent_health(component_health=False, last_error=f"Opentelemetry SDK require Python in version 3.8 or higher, [{python_version}] not supported",
```

Comment:
should we rename the function to so it's clear that it is used only for `UnsupportedRuntimeVersion` status?